CMakeLists.txt

```diff
@@ -7,22 +7,55 @@ cmake_minimum_required(VERSION 3.6)
 # Note: CMake is mainly meant for usage under Windows, while under Unix we
 # prefer to use autotools; hence assume for Unix this file has quirks.
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 file(
      GLOB
      sources
      "src/libmeasurement_kit/*/*.c"
      "src/libmeasurement_kit/*/*.cpp"
     )
 
+add_definitions(-DMK_CA_BUNDLE="mk-ca-bundle.pem")
```

Comment:
This is to tell the compiled code where to look for a CA bundle. We can change this in the future, but for now it works well having `measurement_kit.exe` and the CA bundle in the same directory.